### ISSUE TITLE
Training and validation loss graphs

### ISSUE THREAD

**Issue by sahrudaibond** [mvirgo/MLND-Capstone#29]
Please post the training loss graph and validation loss graph for your trained model. I am using as a reference for my dissertation. It will be very helpful.

**Comment by mvirgo** [comment]
My apologies for not getting back to you on this - unfortunately, these graphs are lost to the sands of time, as I did not plot them out originally.

**Comment by mvirgo** [comment]
Closing out since no further comments.
